starlingx/inventory/v1/storagebackends/requests.go

```diff
@@ -16,6 +16,7 @@ type StorageBackendOpts struct {
 	State        *string                 `json:"state,omitempty" mapstructure:"state"`
 	Task         *string                 `json:"task,omitempty" mapstructure:"task"`
 	Services     *string                 `json:"services,omitempty" mapstructure:"services"`
+        Network      *string                 `json:"network,omitempty" mapstructure:"network"`
```

Comment:
Why does this look over-indented - is that showing wrong in github? Did you run the "go fmt" step as described in the developer workflow?
https://github.com/Wind-River/cloud-platform-deployment-manager/blob/master/DEVELOPER.md#development-workflow

Reply:
Didn't run the 'go fmt' step.
Done.
'go vet' and 'go test' fail unrelated to this commit, at certificates component.

Reply:
Good to know. Presumably an expired test certficate.